cg/cli/downsample.py

```diff
@@ -24,13 +24,15 @@ def downsample():
     "samples",
     help="Downsample reads in one or multiple samples in a case. Usage: \n"
     "For a single sample: cg downsample samples -c supersonicturtle -cn new_case_name -i ACC1234 0.1\n"
-    "For multiple samples:cg downsample samples -c supersonicturtle -cn new_case_name -i ACC1234 0.1 -i ACC12324 10",
+    "For multiple samples:cg downsample samples -c supersonicturtle -cn new_case_name -i ACC1234 0.1 -i ACC12324 10\n"
+    "If multiple samples are provided possible sample relationships will need to be added using the:\n"
+    "cg add relationship command.",
```

Comment:
feel free to disregard
```suggestion
    "If multiple samples are provided possible sample relationships will need to be added using:\n"
    "cg add relationship.",
```